### PR TITLE
4 introduction

### PR DESCRIPTION
**Pull request by filippopedrazzinifp**
- Completed main concepts section in introduction
- Added links to dev overview and service packaging overview

### CODE REVIEW

File: docs/intro.md [hunk 1]
```diff
@@ -16,16 +16,15 @@ Prem is a platform for running open-source AI models on your own devices, either
...
-
+- **Prem Interface**: An interface is the combination of input and output for a Prem Service. It is a semantic type that defines the structure of the data that can be sent to the model and the structure of the data that the model will return. Prem comes with a set of standard interfaces, but you can also create your own. Currently the following interfaces have been created: 😃 Chat, 📕 Embeddings, 🏛️ Vector Store, 🎨 Diffuser. More to come.
 - [**Prem App**](/docs/prem-app/installation/install-macos-app.md): the desktop app for MacOS that allows you to run AI models on your own computer.
-
-- [**Prem Daemon**](#): the daemon that runs on your machine and allows you to run AI models on your own infrastructure
-
-- [**Prem Registry**](#): Each Prem Service can be published on the public list of AI services.
\ No newline at end of file
+- **Prem Daemon**: the daemon that runs on your machine and allows you to run AI models on your own infrastructure. I trepresents the component which exposes all the endpoints necessary to handle the different services and the underlying infrastructure.
```
Comment (tiero) [inline docs/intro.md:29]:
```suggestion
- **Prem Daemon**: the daemon that runs on your machine and allows you to run AI models on your own infrastructure. It represents the component that exposes all the endpoints necessary to handle the different services and the underlying infrastructure.
```

File: docs/registry.md [hunk 2]
```diff
@@ -2,3 +2,65 @@
...
+
+### 1. Test
+
+Before submitting the service, make sure that the image has been tested with the corresponding Prem Interface. 
+
+> Development Mode To Be Defined. An Idea could be provide external URL in settings.
+
+### 2. Submit
+
+When you are done with testing and you are confident that your service is ready for Prem, you can fork https://github.com/premAI-io/prem-registry/ and add a pull request against `dev` branch. Here the folders and files necessary for the pull request.
```
Comment (tiero) [inline docs/registry.md:22]: why not main?

File: docs/registry.md [hunk 3]
```diff
@@ -2,3 +2,65 @@
...
+        },
+        "defaultPort": 8000, // try to keep 8000
+        "defaultExternalPort": 8333 // put a random number here
+    }
+    ```
+    
+    - `logo.svg`: a logo of your service in svg format 256x256.
+    - `README.md`: documentation of the service, how to use it, tips and tricks, license. Provide all the insights and the information necessary to exploit the model at maximum capacity.
+
+### 3. Deploy 🎉
```
Comment (tiero) [inline docs/registry.md:58]:
```suggestion
### 3. 🚀 Submit 
```

File: docs/registry.md (hunk 2 above)
Comment (filippopedrazzinifp) [reply docs/registry.md:22]: main is stable, dev has one step more. Prem team tests on dev. We can change the behaviour, but right now it works in this way.

File: docs/registry.md (hunk 3 above)
Comment (filippopedrazzinifp) [reply docs/registry.md:58]: The previous section is called submit already

**Pull request merged**